src/index.html

```diff
@@ -2,16 +2,169 @@
+            minlength="5"
+            maxlength="12"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            id="name"
+            autocomplete="off">
+        </div>
+
+        <div class="form-field year">
+          <label for="years">How old are You?</label>
+          <input
+            type="number"
+            id="years"
+            value="12"
+            max="100">
+        </div>
+
+        <div class="form-field">
+          <label for="Data">Full date of birth:</label>
+          <input type="date" id="Data">
+        </div>
+
+        <div class="form-field">
+          <label for="agreement">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            id="agreement">
+        </div>
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration</legend>
+
+        <div class="form-field">
+          <label for="Email">E-mail:</label>
+          <input
+            type="email"
+            id="Email"
+            placeholder="email@example.com">
+        </div>
+
+        <div class="form-field">
+          <label for="Password">Password:</label>
+          <input
+            type="password"
+            id="Password">
+        </div>
+      </fieldset>
+
+
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          <span>Do you love cats?</span>
+          <input
+            id="Yes"
+            type="radio"
+            name="cats"
+            value="Yes">
+          <label for="Yes">Yes</label>
+          <input
+            id="No"
+            type="radio"
+            name="cats"
+            value="No">
+          <label for="No">No</label>
+        </div>
+
+        <div class="form-field">
+          <label for="Color">What is your favorite color?</label>
+          <input type="color" id="Color">
+        </div>
+
+        <div class="form-field">
+          <label for="TimeToSleep">What time do you go to bed?</label>
+          <input
+            type="time"
+            id="TimeToSleep"
+            step="2">
+        </div>
+
+        <div class="form-field">
+          <label for="Cars">What are your favorite brand of cars?</label>
+          <select
+            name="Cars"
+            id="Cars"
+            required
+            multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </div>
+
+        <div class="form-field">
+          <label for="Work">How do you rate our work?</label>
+          <input
+            id="Work"
+            type="range"
+            min="1"
+            max="10"
+            value="1">
+        </div>
+      </fieldset>
+
+
+      <fieldset>
+        <legend>Additional info</legend>
```

Comment:
```suggestion
      </fieldset>

      <fieldset>
        <legend>Additional info</legend>
```
one blank line maximum **ALL similar places**